installation.md

```diff
@@ -130,26 +126,30 @@ You can now exit `psql`, by pressing `<Ctrl-D>` or typing `\q<Enter>`.
 
 ### Making the Postgres Credentials Available in Your Environment
 
-You will see in `config/database.yml` that the Postgres username and password are fetched from the environment variables `PG_USERNAME` and `PG_PASSWORD`, respectively. (The production environment password is an exception, coming from `DIAPER_DATABASE_PASSWORD`.) You'll need to make this happen, for example by putting them in a startup script, or (quick and dirty) typing `export` commands in the terminal:
+You will see in `config/database.yml` that the Postgres username and password are fetched from the environment variables `PG_USERNAME` and `PG_PASSWORD`, respectively. (The production environment password is an exception, coming from `DIAPER_DATABASE_PASSWORD`.) The easiest way to set these variables is by copying the `.env.example` file to `.env` and then editing it:
+
+`cp .env.example .env`
 
-For OS X users, remember to replace `postgres`, below, with the user name you used in psql.
+Now edit the top two lines of this file. For OS X users, remember to replace `postgres`, below, with the user name you used in psql.
 
 ```
-export PG_USERNAME=postgres
-export PG_PASSWORD=password
+PG_USERNAME=postgres
+PG_PASSWORD=password
 ```
 
 ### Initializing the Data Base
 
 From your project directory, initialize the data base:
 
-`rails db:setup`
-
+`rails db:setup db:migrate db:test:prepare db:seed`
 
 ### Testing the Application
 
 You should now be able to successfully run the application:
 
-`rails server`
+```
+bin/webpack-dev-server
+rails server
+```
```

Comment:
did you mean to include these modifications in this PR?

Reply:
I did (since I added them as I went) but I can definitely take it out and move it to another PR if you like.